automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -2,21 +2,30 @@ package org.broadinstitute.dsde.workbench.leonardo.notebooks
 
 import cats.effect.unsafe.implicits.global
 import org.broadinstitute.dsde.workbench.auth.AuthToken
-import org.broadinstitute.dsde.workbench.leonardo.{LeonardoConfig, RuntimeFixtureSpec, SSH}
+import org.broadinstitute.dsde.workbench.leonardo.{LeonardoApiClient, LeonardoConfig, RuntimeFixtureSpec, SSH}
 import org.broadinstitute.dsde.workbench.leonardo.RuntimeFixtureSpec.runtimeFixtureZone
-import org.scalatest.DoNotDiscover
+import org.broadinstitute.dsde.workbench.leonardo.runtimes.RuntimeGceSpecDependencies
+import org.scalatest.{DoNotDiscover, ParallelTestExecution}
+import cats.syntax.all._
+import org.broadinstitute.dsde.workbench.ResourceFile
+import org.broadinstitute.dsde.workbench.leonardo.BillingProjectFixtureSpec.workspaceNameKey
 
 /**
  * This spec verifies notebook functionality specifically around the Python 3 kernel.
  */
 @DoNotDiscover
-class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
+class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils with ParallelTestExecution {
```

Comment:
would this be renamed since we are including two different specs now?

Reply:
Yeah, I'll rename it since we seem ok with consolidating it into one spec